pkg/controller/migmigration/backup.go

```diff
@@ -11,28 +11,32 @@ import (
 	velero "github.com/heptio/velero/pkg/apis/velero/v1"
 	"github.com/pkg/errors"
 	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
 	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 var resticPodPrefix = "restic-"
+var rhel7ImageRef = "registry.access.redhat.com/rhel7"
 
-// Ensure the backup on the source cluster has been created
-// and has the proper settings.
-func (t *Task) ensureBackup() error {
-	newBackup, err := t.buildBackup()
+// Ensure the initial backup on the source cluster has been created and has the
+// proper settings.
+func (t *Task) ensureInitialBackup() error {
+	includeClusterResources := false
+	newBackup, err := t.buildBackup(&includeClusterResources)
 	if err != nil {
 		log.Trace(err)
 		return err
 	}
-	foundBackup, err := t.getBackup()
+	delete(newBackup.Annotations, MigQuiesceAnnotationKey)
```

Comment:
Yeah so this is definitely temporary as I discussed with you and Scott. Ideally the plugin doesn't act on quiesce annotations.

But I need to delete this currently because if we were to quiesce on the first backup (and by that I mean set the annotation with the current plugin), then the pod will go down and we won't be able to annotate it and create a stage pod in before the next backup.